Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -87,11 +99,44 @@ - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVide
     return nil;
 }
 
+- (BOOL)enableVideoSending:(BOOL)enable forCall:(OCTCall *)call error:(NSError **)error
+{
+    OCTToxAVVideoBitRate bitrate = (enable) ? kDefaultVideoBitRate : kOCTToxAVVideoBitRateDisable;
+    BOOL success = [self setVideoBitrate:bitrate forCall:call error:error];
+
+    if (! success) {
+        return NO;
+    }
+
+    if (enable && (! [call isPaused])) {
+        OCTFriend *friend = [call.chat.friends firstObject];
+        self.videoEngine.friendNumber = friend.friendNumber;
+        [self.videoEngine startSendingVideo];
+    }
+    else {
+        [self.videoEngine stopSendingVideo];
```

Comment:
Hmm I'm going to have to revisit this later. Tried to change it but then I had some trouble figuring out how to mock audioEngine since I needed to stub `[self.audioEngine isAudioRunning:nil]` to both YES and NO in one method.